guides/micronaut-metrics-oci/metadata.json

```
@@ -0,0 +1,13 @@
+{
+  "base": "micronaut-metrics",
+  "title": "Collect Metrics with Micronaut and Monitor them on Oracle Cloud",
```

```suggestion
  "title": "Collect Metrics with the Micronaut Framework and Monitor them on Oracle Cloud",
```

see https://micronaut.io/brand-guidelines/